packages/sdks/snippets/angular/src/app/editable-regions/custom-columns/custom-columns.component.ts

```diff
@@ -15,45 +15,65 @@ import type {
   standalone: true,
   imports: [CommonModule, Blocks],
   template: `
-    <div
-      style="display: flex; flex-direction: row; justify-content: space-around; border: 10px solid #ccc; padding: 10px"
-    >
-      <blocks
-        [blocks]="columns[0].blocks"
-        [path]="'component.options.columns.0.blocks'"
-        [parent]="builderBlock.id"
-        [context]="builderContext"
-        [registeredComponents]="builderComponents"
-      ></blocks>
-      <blocks
-        [blocks]="columns[1].blocks"
-        [path]="'component.options.columns.1.blocks'"
-        [parent]="builderBlock.id"
-        [context]="builderContext"
-        [registeredComponents]="builderComponents"
-      ></blocks>
-    </div>
+    <blocks
+      [blocks]="column1.blocks"
+      [path]="'component.options.column1.blocks'"
+      [parent]="builderBlock.id"
+      [context]="builderContext"
+      [registeredComponents]="builderComponents"
+      [styleProp]="customStyles"
+    />
+    <blocks
+      [blocks]="column2.blocks"
+      [path]="'component.options.column2.blocks'"
+      [parent]="builderBlock.id"
+      [context]="builderContext"
+      [registeredComponents]="builderComponents"
+      [styleProp]="customStyles"
+    />
   `,
 })
 export class CustomColumnsComponent {
   @Input() builderBlock!: BuilderBlock;
-  @Input() columns: { blocks: BuilderBlock[] }[] = [];
-  @Input() builderComponents: RegisteredComponents = {};
+  @Input() column1!: { blocks: BuilderBlock[] };
+  @Input() column2!: { blocks: BuilderBlock[] };
+  @Input() builderComponents!: RegisteredComponents;
   @Input() builderContext!: BuilderContextInterface;
+
+  @HostBinding('style') customStyles = {
+    display: 'flex',
+    flexDirection: 'row',
+    gap: '10px',
+    padding: '20px',
+    justifyContent: 'space-around',
+    border: '1px solid rgba(26, 115, 232, 1)',
+  };
```

Comment:
I would remove all of these styles completely, even if it makes the component look slightly off.

Reply:
Done. I will push it now.